sbnana/CAFAna/Systs/EnergySysts.h

```diff
@@ -0,0 +1,240 @@
+                                       EnergyScaleSystParticle::Muon>    kEnergyScaleMuonSqrtUBBig;
+  extern const EnergyScaleSystUncorrUB<EnergyScaleSystTerm::InverseSqrt, 
+                                       EnergyScaleSystParticle::Muon>    kEnergyScaleMuonInvSqrtUBBig;
+
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::Constant, 
+                                       EnergyScaleSystParticle::Muon>    kEnergyScaleMuonFDBig;
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::Sqrt, 
+                                       EnergyScaleSystParticle::Muon>    kEnergyScaleMuonSqrtFDBig;
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::InverseSqrt, 
+                                       EnergyScaleSystParticle::Muon>    kEnergyScaleMuonInvSqrtFDBig;
+
+  extern const EnergyScaleSystCorr<EnergyScaleSystTerm::Constant, 
+                                   EnergyScaleSystParticle::Hadron>      kEnergyScaleHadronBig;
+  extern const EnergyScaleSystCorr<EnergyScaleSystTerm::Sqrt, 
+                                   EnergyScaleSystParticle::Hadron>      kEnergyScaleHadronSqrtBig;
+  extern const EnergyScaleSystCorr<EnergyScaleSystTerm::InverseSqrt, 
+                                   EnergyScaleSystParticle::Hadron>      kEnergyScaleHadronInvSqrtBig;
+
+  extern const EnergyScaleSystUncorrND<EnergyScaleSystTerm::Constant, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronNDBig;
+  extern const EnergyScaleSystUncorrND<EnergyScaleSystTerm::Sqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronSqrtNDBig;
+  extern const EnergyScaleSystUncorrND<EnergyScaleSystTerm::InverseSqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronInvSqrtNDBig;
+
+  extern const EnergyScaleSystUncorrUB<EnergyScaleSystTerm::Constant, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronUBBig;
+  extern const EnergyScaleSystUncorrUB<EnergyScaleSystTerm::Sqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronSqrtUBBig;
+  extern const EnergyScaleSystUncorrUB<EnergyScaleSystTerm::InverseSqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronInvSqrtUBBig;
+
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::Constant, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronFDBig;
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::Sqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronSqrtFDBig;
+  extern const EnergyScaleSystUncorrFD<EnergyScaleSystTerm::InverseSqrt, 
+                                       EnergyScaleSystParticle::Hadron>  kEnergyScaleHadronInvSqrtFDBig;
+
+  // Vector of energy scale systematics
+  struct EnergySystVector: public std::vector<const ISyst*>
```

Comment:
Does this do anything? I think it has been copied from somewhere else where it provides useful conversion behaviours, but is unneeded here.